src/bloom/BloomFilter.ts

```diff
@@ -34,16 +53,41 @@ export class BloomFilter extends AlphaFilter
     public static readonly DEFAULT_OPTIONS: BloomFilterOptions = {
         strength: { x: 2, y: 2 },
         quality: 4,
+        resolution: 1,
         kernelSize: 5
     };
 
     private _blurXFilter: BlurFilterPass;
     private _blurYFilter: BlurFilterPass;
     private _strength: PointData;
 
-    constructor(options?: BloomFilterOptions)
+    constructor(options?: BloomFilterOptions);
+    /**
+    * @deprecated since 8.0.0
```

Comment:
This Filters project doesn't follow the same major version as the core project. The next major version here is 6.0.0, not 8.0.0.